app/views/admin/showcase_items/_form.html.erb

```diff
@@ -0,0 +1,75 @@
+<%= simple_form_for [:admin, @item] do |f| %>
+  <div class="form-group">
+    <label>Name</label>
```

Comment:
These labels are detached from their form controls, which hurts accessibility. We should either manually add the `for="showcase_item_name"` attribute here (and on the others) or use the built-in labels. You can pass custom label text to these like this if you want:

```erb
<%= f.input :name, label: "Name <small><i>(Administrator reference only; will not appear anywhere.)</i></small>".html_safe %>
```

Reply:
I must have not seen this comment the other day but this has now been addressed. I was following the pattern the other admin interfaces set but you're right.